doc/modules/classes.rst

```diff
@@ -665,6 +665,12 @@ Plotting
 
 .. currentmodule:: sklearn
 
+.. autosummary::
+   :toctree: generated/
+   :template: class.rst
+
+   inspection.PartialDependenceDisplay
```

Comment:
`inspection.PartialDependencePlotDisplay`?

Reply:
why?

Reply:
My bad, we call the function `partial_dependence` and not `partial_dependence_plot`